Frends.Community.GoogleApi.Tests/Frends.Community.GoogleApi.Tests.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net471</TargetFrameworks>

```

Comment:
Shouldn't the target be same for tests and the project?

Reply:
Unit tests usually target only one target. Having two targets seem to cause some issues in VS.